common/djangoapps/student/models.py

```diff
@@ -273,7 +315,7 @@ def email_exists_or_retired(email, check_for_new_site=False):
             exists = current_org.userorganizationmapping_set.filter(user__email=email).exists()
     else:
         exists = User.objects.filter(email=email).exists()
-    return exists or is_email_retired(email)
+    return exists or is_email_retired(email, check_within_organization=not check_for_new_site)
```

Comment:
Thanks for the comment. How does it look now @ahmedaljazzar?

```suggestion
    check_within_organization = not check_for_new_site # Allow learners to spin their new Tahoe trial
    return exists or is_email_retired(email, check_within_organization=check_within_organization)
```